### PR TITLE
R112 merge in 52982 branded password reset pages

### PR DESCRIPTION
**Pull request by phudson-he**
To include the branding changes to all the password reset page, matching the branded landing and login pages

### CODE REVIEW

File: arches_her/templates/registration/password_reset_complete.html [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+
+{% block content %}
+
+<form id="login-form">
+	<fieldset>
+		<legend class="arches-signin-subtext">
+			{% trans "Password Reset" %}
+		</legend>
+        <div class="form-group">
+            <p>Your password has been set.  You may go ahead and <a href="{% url 'auth' %}">log in now</a>.</p>
```
Comment (RobOatesHistoricEngland) [inline arches_her/templates/registration/password_reset_complete.html:21]: That anchor link looks a little odd to me, especially when it's proceeded by `log in now` with no spacing. Is this right?

File: arches_her/templates/registration/password_reset_confirm.html [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+			<div class="form-group">
+				{{ form.new_password2.errors }}
+				{{ form.new_password2 }}
+			</div>
+			<div class="form-group">
+				<button id="sign-in-button" class="btn btn-primary btn-lg" type="submit">{% trans "Set new password" %}</button>
+			</div>
+		{% else %}
+			<div class="form-group">
+				<p>The password reset link was invalid, possibly because it has already been used.  Please <a href="{% url 'password_reset' %}">request a new password reset</a>.</p>
```
Comment (RobOatesHistoricEngland) [inline arches_her/templates/registration/password_reset_confirm.html:28]: Same comment here about the anchor link and the text immediately following it.

**Review by RobOatesHistoricEngland — CHANGES_REQUESTED** [review]

File: arches_her/templates/registration/password_reset_complete.html (hunk 1 above)
Comment (phudson-he) [reply arches_her/templates/registration/password_reset_complete.html:21]: Renders fine...
image

File: arches_her/templates/registration/password_reset_confirm.html (hunk 2 above)
Comment (phudson-he) [reply arches_her/templates/registration/password_reset_confirm.html:28]: Also renders fine...
image

**Review by RobOatesHistoricEngland — APPROVED** [review]
Looks good

**Review by Rehaan-Arif — APPROVED** [review]
tested successfully

File: arches_her/templates/registration/password_reset_email.html [hunk 3]
```diff
@@ -0,0 +1,12 @@
+{% load i18n %}{% autoescape off %}
+{% blocktrans %}You're receiving this email because you requested a password reset for your user account at {{ site_name }}.{% endblocktrans %}
+
+{% trans "Please go to the following page and choose a new password:" %}
+{% block reset_link %}
+{{ protocol }}://{{ domain }}{% url 'password_reset_confirm' uidb64=uid token=token %}
+{% endblock %}
+{% trans "Your username, in case you've forgotten:" %} {{ user.get_username }}
```
[the remark this answers is not in the document]
Comment (phudson-he) [reply arches_her/templates/registration/password_reset_email.html]: Line removed

File: arches_her/templates/change_password.htm [hunk 4]
```diff
@@ -0,0 +1,58 @@
...
+                    <div class="form-group">
+                        <fieldset>
+                            <legend>
+                                Enter your old password, new password and confirm new password form fields to change your password
+                            </legend>
+                            <div class="form-group">
+                                <label class="reqlabel control-label widget-input-label" for="old_password">Old Password (required)</label>
+                                <div class="input-group">
+                                    <input id="old_password" type="password" class="form-control input-lg" data-bind="textInput:credentials.old_password" name='old_password' aria-required="true">
+                                    <div class="error-message" data-bind="text:invalidPassword"></div>
```
[the remark this answers is not in the document]
Comment (phudson-he) [reply arches_her/templates/change_password.htm]: Added aria-live attribute, errors now read out by screen reader, checked with NVDA

File: arches_her/templates/change_password.htm [hunk 5]
```diff
@@ -0,0 +1,58 @@
...
+                                </div>
+                            </div>
+                            <div class="form-group">
+                                <label class="reqlabel control-label widget-input-label" for="new_password1">New Password (required)</label>
+                                <div class="input-group">
+                                    <input id="new_password1" type="password" class="form-control input-lg" data-bind="textInput:credentials.new_password" name='new_password1' aria-required="true">
+                                    <div class="error-message-container" data-bind="foreach: validationErrors">
+                                        <div data-bind="text: $data" class="error-message"></div>
```
[the remark this answers is not in the document]
Comment (phudson-he) [reply arches_her/templates/change_password.htm:27]: Added aria-live attribute, errors now read out by screen reader, checked with NVDA

**Review by aj-he — APPROVED** [review]
Looks good to me.

**Pull request merged**